src/libstd/primitive_docs.rs

```diff
@@ -67,6 +67,123 @@
 #[stable(feature = "rust1", since = "1.0.0")]
 mod prim_bool { }
 
+#[doc(primitive = "never")]
+//
```

Comment:
Why this line?

Reply:
I'm not sure why it was there in the first place, but it does match the other primitive docs.

Reply:
We should remove them then!

@rust-lang/cleanup-team